packages/manager/apps/catalog-revamp/src/utils/utils.ts

```diff
@@ -0,0 +1,89 @@
+export type Product = {
+  id: number;
+  name: string;
+  description: string;
+  lang: string;
+  categories: string[];
+  url: string;
+  regionTags: string[];
+  productName: string;
+  order: string;
+  universe: string;
+  category: string;
+};
+
+type ProductKey = 'universe' | 'category' | 'name';
+
+export const groupItems = (itemsToGroup: Product[]) => {
```

Comment:
groupItemsByUniverse would be a better name isn't it ?

Reply:
Done ✅